Dockerfile.agent

```diff
@@ -4,15 +4,16 @@ MAINTAINER Kontena, Inc. <info@kontena.io>
 RUN apk update && apk --update add tzdata ruby ruby-irb ruby-bigdecimal \
     ruby-io-console ruby-json ca-certificates libssl1.0 openssl libstdc++
 
-ADD Gemfile /app/
-ADD Gemfile.lock /app/
+ADD common/kontena-common.gemspec /opt/kontena/common/
+ADD agent/Gemfile agent/Gemfile.lock /opt/kontena/agent/
 
 RUN apk --update add --virtual build-dependencies ruby-dev build-base openssl-dev && \
     gem install bundler --no-ri --no-rdoc && \
-    cd /app ; bundle install --without development test && \
+    cd /opt/kontena/agent/ ; bundle install --without development test && \
     apk del build-dependencies
 
-WORKDIR /app
-ADD . /app
+ADD common /opt/kontena/common
+ADD agent /opt/kontena/agent
 
-CMD ["/app/bin/kontena-agent"]
+WORKDIR /opt/kontena/agent
+CMD bundle exec /opt/kontena/agent/bin/kontena-agent
```

Comment:
Using `bundle exec` to add the `/opt/kontena/common/lib` to the `$LOAD_PATH`... not sure if the best way to do it, but probably the fastest.

* `kontena gem build /opt/kontena/common/kontena-common.gemspec && gem install kontena-common-*.gem`?
* ???